test/performance/playwright.config.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import os from 'os';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { defineConfig, devices } from '@playwright/test';
+
+process.env.WP_ARTIFACTS_PATH ??= path.join( process.cwd(), 'artifacts' );
+process.env.STORAGE_STATE_PATH ??= path.join(
+	process.env.WP_ARTIFACTS_PATH,
+	'storage-states/admin.json'
+);
+process.env.ASSETS_PATH = path.join( __dirname, 'assets' );
+
+const config = defineConfig( {
+	reporter: process.env.CI
+		? undefined // We're using another reporter in CI.
+		: [ [ 'list' ], [ './config/performance-reporter.ts' ] ],
+	forbidOnly: !! process.env.CI,
+	fullyParallel: false,
+	workers: 1,
+	retries: 0,
+	timeout: parseInt( process.env.TIMEOUT || '', 10 ) || 100_000, // Defaults to 100 seconds.
+	testDir: fileURLToPath( new URL( './specs', 'file:' + __filename ).href ),
+	outputDir: path.join( process.env.WP_ARTIFACTS_PATH, 'test-results' ),
+	snapshotPathTemplate:
+		'{testDir}/{testFileDir}/__snapshots__/{arg}-{projectName}{ext}',
+	globalSetup: fileURLToPath(
+		new URL( './config/global-setup.ts', 'file:' + __filename ).href
+	),
+	use: {
+		baseURL: process.env.WP_BASE_URL || 'http://localhost:8889',
+		headless: true,
+		viewport: {
+			width: 960,
+			height: 700,
+		},
+		ignoreHTTPSErrors: true,
+		locale: 'en-US',
+		contextOptions: {
+			reducedMotion: 'reduce',
```

Comment:
I wonder if this has any performance impact? I'm also not sure if the original test is using this or not 😅 .

Reply:
It might have, though [we do also use it with Puppeteer](https://github.com/WordPress/gutenberg/blob/89df866f4c581c3f4c8109e3e8c655263b3840e5/packages/e2e-tests/config/setup-performance-test.js#L25). I'd leave it as is for the sake of consistency and explore the impact in a follow-up PR.